owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -67,6 +78,8 @@ import timber.log.Timber
  * classes
  */
 class MainApp : Application() {
+    private val KEY_LAST_SEEN_VERSION_CODE = "lastSeenVersionCode"
```

Comment:
I noticed that you have the same `KEY` in different classes, what happens if someone tries to change the name of the `variable` or its `value`? I think you should centralize it in a class (for example in a `companion object`) and call it always. What do you think?